projects/example-app/src/app/books/components/book-search.component.ts

```diff
@@ -11,7 +11,7 @@ import { Component, Output, Input, EventEmitter } from '@angular/core';
             matInput
             placeholder="Search for a book"
             [value]="query"
-            (keyup)="search.emit($event.target.value)"
+            (keyup)="search.emit($any($event.target).value)"
```

Comment:
it would be cool to not to have to cast to `any` here... but maybe such is life with JS events. but I coulda sworn there was sufficient typing for a `KeyboardEvent`.